registry/src/pages/home.js

```diff
@@ -1,53 +1,53 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Container, InputGroup, FormControl } from "react-bootstrap";
+import { searchPackage, setQuery } from "../store/actions/searchActions";
 
 import "../home.css";
 
 const Home = () => {
   return (
-    <Container className="home-container">
-      <h2>Fortran</h2>
-      <h2>High-performance parallel programming language</h2>
-      <div className="features-list">
-        <h2 className="heading-features">Features</h2>
-        <h4>High Performance</h4>
-        <p>
-          Fortran has been designed from the ground up for computationally
-          intensive applications in science and engineering. Mature and
-          battle-tested compilers and libraries allow you to write code that
-          runs close to metal, fast.
-        </p>
-        <h4>Statically and strongly typed</h4>
-        <p>
-          Fortran is statically and strongly typed, which allows the compiler to
-          catch many programming errors early on for you. This also allows the
-          compiler to generate efficient binary code.
-        </p>
-        <h4>Easy to learn and use</h4>
-        <p>
-          Fortran is a relatively small language that is surprisingly easy to
-          learn and use. Expressing most mathematical and arithmetic operations
-          over large arrays is as simple as writing them as equations on a
-          whiteboard.
-        </p>
-        <h4>Versatile</h4>
-        <p>
-          Fortran allows you to write code in a style that best fits your
-          problem: imperative, procedural, array-oriented, object-oriented, or
-          functional.
-        </p>
-        <h4>Natively parallel</h4>
-        <p>
-          Fortran is a natively parallel programming language with intuitive
-          array-like syntax to communicate data between CPUs. You can run almost
-          the same code on a single CPU, on a shared-memory multicore system, or
-          on a distributed-memory HPC or cloud-based system. Coarrays, teams,
-          events, and collective subroutines allow you to express different
-          parallel programming patterns that best fit your problem at hand.
-        </p>
+    <Container id="home-container">
+      <div id="fpm-logo">
+        <img src="https://raw.githubusercontent.com/fortran-lang/assets/main/fpm/logo/2-color-alt/png/full-color-alt.png" />
       </div>
+
+      <HomeSearchField type="text" placeholder="Search package" />
+      <p id="fpm-subscript">The official registry for fpm packages</p>
     </Container>
   );
 };
 
 export default Home;
+
+function HomeSearchField(props) {
+  const query = useSelector((state) => state.search.query);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const search = () => {
+    if (query.trim().length !== 0) {
+      dispatch(searchPackage(query, 0));
+      navigate("/search");
+    }
+  };
+
+  const handleKeyDown = (event) => {
```

Comment:
This function must be embedded in both the home and the Navbar, for the search to work by enter Key.

Reply:
You have already done it here https://github.com/fortran-lang/registry/pull/22/commits/8b167ada37b2d0d79a093eb99e86b5f4f6725adf and that has been merged already. So, I don't think I will have to do it again